src/run/editor/mod.rs

```diff
@@ -539,7 +536,7 @@ impl Editor {
             return Ok(());
         }
 
-        if validate_comparison_name(&self.run, new) {
+        if !new.starts_with("[Race]") && !self.run.comparisons().any(|c| c == new) {
```

Comment:
I'd like to have that logic in a single place. This is a copy of what's in add_custom_comparison. Maybe keep a validate_comparison_name somewhere and use it in both places.